include/mbgl/map/map.hpp

```diff
@@ -41,7 +41,7 @@ class Map : private util::noncopyable {
                  GLContextMode contextMode = GLContextMode::Unique,
                  ConstrainMode constrainMode = ConstrainMode::HeightOnly,
                  ViewportMode viewportMode = ViewportMode::Default,
-                 const optional<std::string>& programCacheDir = {});
+                 const optional<std::string>& programCacheDir = optional<std::string>());
```

Comment:
GCC 4.9 is unable to find an appropriate ctor when having auto-declared params :sob: